backend/routes/subscribe.js

```
@@ -1,45 +1,69 @@
 const express = require("express");
+const express = require("express");
```

Biome (2.1.2)

[error] Shouldn't redeclare 'express'. Consider to delete it or rename it.

'express' is defined here:


(lint/suspicious/noRedeclare)